jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -210,7 +210,9 @@ public static void runRunnableInNotCheckpointSafeMode(Runnable run) {
 
 	@SuppressWarnings("restriction")
 	private static Unsafe unsafe;
+	/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
This file contains other security manager-related API usages, such as `AccessController.doPrivileged()`. Are they to be addressed in another PR?

Reply:
Yes I am planning to do those separately since there are also many uses.